lib/commands/actions.js

```diff
@@ -13,6 +13,8 @@ const dragStepsPerSec = 40;
 const CONTAINER_PATH_MARKER = '@';
 // https://regex101.com/r/PLdB0G/2
 const CONTAINER_PATH_PATTERN = new RegExp(`^${CONTAINER_PATH_MARKER}([^/]+)/(.+)`);
+const ANDROID_SDCARD_PATH_RE = new RegExp('^/mnt/sdcard/');
```

Comment:
can it be simply /sdcard/ thou? The scanning based on target path anyway looks quite unstable to me? Can we, for example, analyse the type of the source file, for example using the system `file` utility (if present) or some node package to identify whether the particular file is a media file?

Reply:
sure, we could do that. or we could assume that if someone is pushing a file to the device it's most likely going to be a media file and just always do the broadcast. it probably doesn't hurt to do the broadcast every time anyway, since it's a relatively instant operation.